Packs/CofenseTriage/Integrations/CofenseTriagev3/README.md

```diff
@@ -0,0 +1,2393 @@
+The Cofense Triage v3 integration uses the Cofense Triage v2 API (previous integrations were limited to Triage v1 API) that allows users to ingest phishing reports as incident alerts and execute commands.
+
+Security teams can ingest data from Triage such as email reporters, email reports and clusters, threat indicators, and rule matching to name a few. In addition, ingest and create threat indicators, categorize reports, and obtain second stage threat indicators from malicious emails. This integration was integrated and tested with version 1.22.0 of Cofense Triage.
+
+## What's new in Cofense Triage v3?
+Use Cofense Triage v3 XSOAR integration application to take advantage of Cofense Triage v2 API for bidirectional communication between Triage and XSOAR. 
+Earlier, the Cofense Triage v2 was only capable of pulling the indicators from Cofense. The new integration provides:
+ - Report Categorization playbook investigates reports that are unprocessed or uncategorized on Cofense Triage as incident alerts in XSOAR and categorizes them based on the severity of the incident.
+ - Cluster Report Categorization playbook is used to retrieve the reports of specific clusters and perform the categorization of reports.
+ - Automation scripts that download all reports associated with the email address and enrich the threat indicators.
+ - Mirroring feature that updates the XSOAR incidents when Cofense Triage reports are updated.
+
+## What Mirroring functionality does?
+This feature is compliant with XSOAR version 6.0 and above. This feature automatically updates the Cortex XSOAR incidents if any report is changed or updated in Cofense Triage. 
```

Comment:
no need to elaborate on the feature here, more on whats actually being mirrored, like what data